pkgs/by-name/xp/xpra-html5/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  python3,
+  uglify-js,
+}:
+stdenv.mkDerivation rec {
```

```suggestion
stdenvNoCC.mkDerivation rec {
```
Also I think `let ... in` is preferred to `rec`.

---

It doesn't really matter